pix4d-dependabot/lib/helpers/path_level.rb

```diff
@@ -1,22 +1,29 @@
 # frozen_string_literal: true
 
+def select_path_per_directory(git_tree, directory)
+  git_tree.select { |f| f.path.include?(directory.to_s) }.
+    select { |f| f.path.include?("Dockerfile") }.
```

Comment:
Couldn't the two `select` be joined in one?
```suggestion
  git_tree.select { |f| f.path.include?(directory.to_s) && f.path.include?("Dockerfile") }.
```

Reply:
Thanks for suggestion. I proposed three solutions. Have a look above.